packages/scrollable-image-ui/src/components/app.js

```diff
@@ -0,0 +1,92 @@
+import AddImageLinkForm from './form'
+import EmbeddedCodeModal from './embedded-code-modal'
+import ImageList from './image-list'
+import React, { useState } from 'react'
+import useImagesState from '../hooks/use-images-state'
+// @material-ui
+import Checkbox from '@material-ui/core/Checkbox'
+import Container from '@material-ui/core/Container'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Typography from '@material-ui/core/Typography'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    marginTop: theme.spacing(3),
+    textAlign: 'center',
+  },
+  formControlLabel: {
+    display: 'block',
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}))
+
+const Content = () => {
+  const classes = useStyles()
+  const { imageLinks, addImageLink, deleteImageLink } = useImagesState([])
+  const [state, setState] = useState({
+    checkedLazyload: false,
+  })
```

Comment:
```
const  [toLazyLoad, setLazyLoad] = useState(false) 
```
?

Reply:
Before having multiple options, it could be updated like this.

It seems tidier.